src/windows/GeolocationProxy.js

```diff
@@ -181,10 +178,11 @@ module.exports = {
             callbacks = ids[clientId];
```

Comment:
Changed the declarations. I am not sure about adding an assertion or throwing an exception. Consider a scenario in which I press the "Stop watching" in mobilespec before "Start Watching". This is a valid scenario. I just don't want an assertion or exception in such a scenario. (as the reason is simply there is nothing to stop)